src/theme.ts

```diff
@@ -29,6 +39,9 @@ export default createTheme({
     MuiContainer: {
       root: {
         width: '950px',
+        [defaultTheme.breakpoints.down(960)]: {
```

Comment:
I tried that, but it threw everything off for some of the screen sizes in between `md` and `lg` for some reason. The screenshot below is what a screen size of `813 x 880` looks if we were to use `.down(768)`. 960 is the smallest `md` size for [MUI's default values](https://v4.mui.com/customization/breakpoints/#breakpoints), and it seems to do the trick for some reason. But I'm sure I can figure something out if we wanted to change it to `.down(768)`, it would just require some more changes in the other files. Let me know what you think!


![Screen Shot 2022-02-02 at 4 42 33 PM](https://user-images.githubusercontent.com/77076398/152261965-6ad4346b-e276-4724-90d2-cdb59797d0f5.png)

Reply:
What about
```
    MuiContainer: {
      root: {
        width: '950px',
        maxWidth: 'calc(100vw - 160px)',
      },
    },
```
with `160px` saved as a theme variable?